types/ably/auto-sni/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for auto-sni v2.1.1
+// Project: https://www.npmjs.com/package/auto-sni
+// Definitions by: Jan Wolf <https://github.com/janwo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace createServer {
+
+}
+declare function createServer(opts: any, app?: any): any;
```

Comment:
The options are [documented](https://github.com/DylanPiercey/auto-sni#example), so they could have a better type than `any`.
Also, the documentation says `Server is a "https.createServer" instance.`, so the return value should be typed too.